analyzers/tests/SonarAnalyzer.TestFramework.Test/Verification/VerifierTest.cs

```diff
@@ -159,9 +159,23 @@ public class Sample
     [DataRow("Dummy.SecondaryLocation.cshtml")]
     public void Verify_RazorWithAdditionalLocation(string path) =>
         DummyWithLocation.AddPaths(path)
+            .WithOptions(ParseOptionsHelper.BeforeCSharp10)
             .WithAdditionalFilePath(AnalysisScaffolding.CreateSonarProjectConfig(TestContext, ProjectType.Product))
             .Verify();
 
+#if NET
```

Comment:
Is this required to not fail in the pipeline?

Reply:
I don't think so, let's try